docs/source/guides/staking_guide.rst

```diff
@@ -237,6 +253,35 @@ the address to checksum format in geth console:
 After this step, you're finished with the Staker, and you can proceed to :ref:`ursula-config-guide`.
 
 
+Manage automatic reward restaking
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+As your Ursula performs work, you can optionally enable the automatic addition of
+all rewards to your existing stake to optimize earnings.  By default this feature is disabled,
+to enable it run:
+
+.. code:: bash
+
+    (nucypher)$ nucypher stake restake --enable
+
+To disable restaking:
+
+.. code:: bash
+
+    (nucypher)$ nucypher stake restake --disable
+
+
+Additionally, you can enable **restake locking**, an on-chain commitment to continue restaking
+until a future period (`release_period`). Once enabled, the `StakingEscrow` contract will not
+allow **restaking** to be disabled until the release period begins, even if you are the stake owner.
```

Comment:
Same question here, is re-stake locking all or nothing?

Reply:
I think it does, yes. If you need partial, I think you split into different nodes